marconi-core/src/Marconi/Core/Transformer/WithAction.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE RankNTypes #-}
+
+{- |
+  A transformer that fans events into an arbitrary action.
+
+ The configuration is a function @Timed (Point event) event -> IO ()@
+
+ See Marconi.Core.Transformer.WithStream.Socket and Marconi.Core.Transformer.WithStream.TBQueue
+ for example usages.
+
+ Once we are close enough to the tip of the chain, `WithCatchup` deactivate itself and pass directly
+ the blocks to the indexer.
```

Comment:
I don't understand this comment. What does that have to do with `WithAction`?

Reply:
Ah my bad, I copy and pasted the structure of `WithCatchup`'s description and must have forgotten to remove the rest!